.github/workflows/do-release.yml

```diff
@@ -15,26 +15,40 @@ on:
 jobs:
   bump-version:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

Comment:
cloning the repository with the app token "should" be enough to modify the files. we will have to test it and if not put back `write` in contents here